src/commands/fun/latex.ts

```diff
@@ -0,0 +1,86 @@
+import { ApplicationCommandOptionData, CommandInteraction, MessageEmbed, MessageAttachment, BaseGuild } from 'discord.js';
+import fetch from 'node-fetch';
+import { createCanvas, loadImage } from 'canvas';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `Accepts a LaTeX expression and posts it as a rendered image.`;
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'input',
+			description: 'The LaTeX expression to render',
+			type: 'STRING',
+			required: true
+		},
+		{
+			name: 'transparent',
+			description: 'Whether to render the image with a transparent background (default: false)',
+			type: 'BOOLEAN',
+			required: false
+		}
+	]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		// input should never be null
+		const tex = interaction.options.getString('input') ?? '';
+		const requestURL = 'https://latex.codecogs.com/svg.json?' + tex;
```

Comment:
```suggestion
		const requestURL = `https://latex.codecogs.com/svg.json?${tex}`;
```
If you have to concatenate a string with a variable, use template literals instead else eslint will whine about it. Plus it makes literals neater, imo :)